modules/apps_api/spec/requests/v0/application_directory_request_spec.rb

```diff
@@ -3,6 +3,37 @@
 require 'rails_helper'
 
 RSpec.describe 'Application Directory Endpoint', type: :request do
+  let(:valid_headers) do
+    { 'Authorization' => Settings.directory.secret }
+  end
```

Comment:
Untested, but something like:
```
let(:authy_auth) { 'super_authentic_string' }
before do
  allow(Settings.directory).to receive(:secret).and_return(authy_auth)
end
let(:valid_headers) do
  { 'Authorization' => authy_auth }
end
```

per our slack discussion.  I would normally test locally before suggesting but I have a few things in progress

Reply:
implemented in latest commit